### DISCUSSION TITLE
New Autospeed Settings Reminder

### DISCUSSION THREAD

**Discussion by Kasiya13** [supermerill/SuperSlicer#1549] [General]
Hi, I just wanted to suggest adding a reminder or something that version 2.3.57 changes some autospeed settings, and that old configs might have some issues. It took me a few hours of fiddling with it to figure out I was using zeros in areas that don't support it anymore.

Thanks for your hard work!

**Comment by supermerill** [comment]
you can open an bug report issue.
what are these fields?
what project give you some problems?
